csharp/tests/Integration/SharedClientTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+using Glide;
```

Comment:
We should consider using a Implicit Global Using statement for Glide, since it will be used in almost all files.
https://learn.microsoft.com/en-us/dotnet/core/project-sdk/overview#implicit-using-directives

Reply:
Fixed in de81cef1f